examples/privatelink-ai-studio/main.tf

```diff
@@ -0,0 +1,238 @@
+terraform {
+  required_version = "~> 1.5"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 3.74"
```

Comment:
use version 3.112.0 -- newer versions have storage account idempotency issues

Reply:
Updated to version 3.112.0.